src/api.js

```diff
@@ -0,0 +1,20 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/uoLjz1rr0ajNeJ1HqpM7/scores/';
+
+async function getScores() {
+  const response = await fetch(url);
+  const data = await response.json();
+  return data.result;
+}
+
+async function addScore(obj) {
```

Comment:
- [ ] Let's use ES6 syntax for our functions as it is a requirement. Kindly change your functions here to arrow functions.